ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -34,7 +34,7 @@ export default function NftsTab({ onAddNFT }) {
   const dispatch = useDispatch();
 
   const { nftsLoading, collections, previouslyOwnedCollection } =
-    useNftsCollections();
+    useNftsCollections(t('nftsPreviouslyOwned'), t('unknownCollection'));
```

Comment:
I don't think we have a translation for `'nftsPreviouslyOwned'`

Reply:
Fixed, that should've been in initial commit.